components/create_post/create_post.jsx

```diff
@@ -796,7 +846,3 @@ export default class CreatePost extends React.Component {
         );
     }
 }
```

Comment:
Could you please add tests to `CreatePost` component?

Reply:
Yes, I was hoping to add test cases was out on a trip. Just catching up with things. I will add test cases first chance I get.